src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const state2 = { ...state };
```

Comment:
It is better to clone the state object inside the loop to ensure that each action is applied to a fresh copy of the state.